privacyidea/lib/config.py

```diff
@@ -61,46 +63,29 @@
 this.config = {}
 
 
-class Singleton(type):
-    """
-    This singleton acts as metaclass for the Caching Objects "ConfigClass"
-    and "PolicyClass".
+class SharedConfigClass(object):
```

Comment:
Please add an underscore to the methods, that should not be used externally:

* clone
* reload_from_db

Reply:
Did this in 55c6374